### PR TITLE
[45] Product Page using HTML

### PR DESCRIPTION
**Pull request by AAOYV2020**
## Task: `Develop a product page for your e-commerce store using HTML Only`
- `Changes I made to given task:
a. Product name
b. Two Product images
c. Product Price
d. Product description
e. An ordered list of similar product names`

### CODE REVIEW

**Comment by TundeOladejo** [comment]
Welldone @AAOYV2020 but your lines of code are not looking too cool. You need to format them properly, removed extra spaces, etc...

File: submissions/anthoniaAdollo/HTML/myproductpage.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+</head>    
+<body>
+    <nav>
+    <a href="#">Home</a>
+    <a href="#"> Product Categories</a>
+    <a href="#">Mommy&Me Blog</a>
+    <a href="#">Contact Us</a>
+
+    </nav>
+
```
Comment (TundeOladejo) [inline submissions/anthoniaAdollo/HTML/myproductpage.html:15]: Please format the whole page properly before we proceed

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/anthoniaAdollo/HTML/myproductpage.html (hunk 1 above)
Comment (AAOYV2020) [reply submissions/anthoniaAdollo/HTML/myproductpage.html:15]: ok, thank you for the feedback. I'll do so now. please hold

Comment (AAOYV2020) [reply submissions/anthoniaAdollo/HTML/myproductpage.html]: Changes have been made, as directed. Please review.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove spaces on your image name. Use `-` or `_` . This task requires that you make use of two images only

**Comment by AAOYV2020** [comment]
@adaugochi thank you for the feedback. Extra images have been removed and image names updated to eliminate spaces. Please review

**Review by adaugochi — APPROVED** [review]

**Pull request merged**